lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/CommunicationGroupDriver.cs

```diff
@@ -42,7 +39,8 @@ namespace Org.Apache.REEF.Network.Group.Driver.Impl
     /// All operators in the same Communication Group run on the the 
     /// same set of tasks.
     /// </summary>
-    public class CommunicationGroupDriver : ICommunicationGroupDriver
+    [Obsolete("Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295 ", false)]
+    public sealed class CommunicationGroupDriver : ICommunicationGroupDriver
```

Comment:
It uses WritableNetworkService that's why I put the obsolete here. Once we change WritableNetworkService to the streaming one then it will no longer be obsolete.